scheduler/cloudformation/cloudformation.go

```diff
@@ -327,8 +326,17 @@ type createStackInput struct {
 // createStack creates a new CloudFormation stack with the given input. This
 // function returns as soon as the stack creation has been submitted. It does
 // not wait for the stack creation to complete.
-func (s *Scheduler) createStack(input *createStackInput, done chan error) error {
-	waiter := s.cloudformation.WaitUntilStackCreateComplete
+func (s *Scheduler) createStack(input *createStackInput, done chan error, ss scheduler.StatusStream) error {
+	waiter := func(input *cloudformation.DescribeStacksInput) error {
+		if err := scheduler.Publish(ss, "Creating stack"); err != nil {
```

Comment:
yea that would be really cool, i set that up on stacker and it was helpful. created an issue to track: https://github.com/remind101/empire/issues/922